runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierPlugin.java

```diff
@@ -331,8 +329,10 @@ public void onGameTick(GameTick event)
 
 		if (client.getGameState() != GameState.LOGGED_IN
 			|| local == null
-			|| System.currentTimeMillis() - client.getMouseLastPressedMillis() < 200
-			|| client.getKeyboardIdleTicks() < 10)
+			// If user has interacted with game in last second then they're not idle so don't send idle notification
+			|| System.currentTimeMillis() - client.getMouseLastPressedMillis() < 1000
+			|| client.getKeyboardIdleTicks() < 50
+			|| client.getMouseIdleTicks() < 50)
```

Comment:
Are you sure about this mouse idle ticks? I seem to recall @deathbeam trying this before and it breaking everything.

Reply:
I have been fishing sacred eels with this version for a few days with no issues, however I haven’t done any other activities. I could remove this to be on the safe side. It’s probably a little overkill in all honesty with the check for clicks and keyboard presses already in there.